src/ccache.c

```diff
@@ -785,13 +785,25 @@ print_included_files(FILE *fp)
 static char *
 make_relative_path(char *path)
 {
+    // if(path[0] == '=') { // Remove first char if string starts with "="
+    //     path++;
+    // }
+
 	if (str_eq(conf->base_dir, "") || !str_startswith(path, conf->base_dir)) {
 		return path;
 	}
 
 #ifdef _WIN32
 	if (path[0] == '/') {
-		path++;  // Skip leading slash.
+		char *p = NULL;
```

Comment:
It looks like you have managed to include the content of 93322f0acf49 in the PR. Please rebase against latest 3.7-maint.